example/gluon/style_transfer/utils.py

```diff
@@ -0,0 +1,119 @@
+import os
+from PIL import Image
+
+import numpy as np
+import mxnet as mx
+import mxnet.ndarray as F
+
+def tensor_load_rgbimage(filename, ctx, size=None, scale=None, keep_asp=False):
```

Comment:
Seem like all these transformations are quite standard, I guess these can be condensed to a couple of lines using mx.image